pyproject.toml

```diff
@@ -71,9 +93,14 @@ known-first-party = ["tap_mysql"]
 convention = "google"
 
 [build-system]
-requires = ["poetry-core==1.9.0"]
-build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core==1.8.1", "poetry-dynamic-versioning"]
```

Comment:
Why downgrade poetry-core?

Reply:
No great reason I was testing, and this worked (it wasn't the cause of the issue I was having). Fixing